app/models/template.rb

```diff
@@ -61,53 +100,28 @@ class Template < ActiveRecord::Base
 
   scope :unarchived, -> { where(archived: false) }
 
-  scope :default, -> { where(is_default: true, published: true).last }
```

Comment:
Moved this to class method. It's not a scope. Scopes should be used to filter records. This method returns a `Template`, not an `ActiveRecord::Relation`